src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1240,18 +1249,21 @@ public void VerifyHMAC_SHA256()
             sign.LoadXml(doc.DocumentElement["Signature"]);
 
             // verify MS-generated signature
-            Assert.True(sign.CheckSignature(new HMACSHA256(hmackey)));
+            Assert.True(sign.CheckSignature(new HMACSHA256(emptyHmacKey)));
```

Comment:
Note: for Verifying signatures I'm still using empty key because it checks it with hardcoded XML and SignXML test does roundtrip checks.